apps/web/components/team/MemberListItem.tsx

```diff
@@ -92,11 +92,11 @@ export default function MemberListItem(props: Props) {
           </div>
           <div className="mt-2 flex ltr:mr-2 rtl:ml-2 sm:mt-0 sm:justify-center">
             {/* Tooltip doesn't show... WHY????? */}
-            {props.member.isMissingSeat && (
+            {/* {props.member.isMissingSeat && (
               <Tooltip side="top" content={t("hidden_team_member_message")}>
                 <TeamPill color="red" text={t("hidden")} />
               </Tooltip>
```

Comment:
We don't need this anymore right? Billing for teams changes based on the number of people automatically

Reply:
It is not implemented yet

Reply:
Can we add context as a note?
```suggestion
            /* @NOTE: not implemented yet */
            {/* {props.member.isMissingSeat && (
              <Tooltip side="top" content={t("hidden_team_member_message")}>
                <TeamPill color="red" text={t("hidden")} />
              </Tooltip>
```